src/org/dotnet/ci/pipelines/scm/VSTSPipelineScm.groovy

```diff
@@ -37,11 +37,11 @@ class VSTSPipelineScm implements PipelineScm {
         job.with {
             // Set up parameters for this job
             parameters {
-                // TODO: VSTS PR params
-                stringParam('sha1', '', 'Incoming sha1 parameter from the GHPRB plugin.')
-                stringParam('GitBranchOrCommit', '${sha1}', 'Git branch or commit to build.  If a branch, builds the HEAD of that branch.  If a commit, then checks out that specific commit.')
-                stringParam('GitRepoUrl', this.getGitUrl(), 'Git repo to clone.')
-                stringParam('GitRefSpec', '+refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*', 'RefSpec.  WHEN SUBMITTING PRIVATE JOB FROM YOUR OWN REPO, CLEAR THIS FIELD (or it won\'t find your code)')
+                stringParam('VSTSCollectionName', repoInfo.collection, 'VSTS collection name')
+                stringParam('VSTSCredentialsId', repoInfo.credentials, 'VSTS credentials id')
+                stringParam('VSTSRepoUrl', this.getGitUrl(), 'VSTS repo to clone.')
+                stringParam('VSTSRefspec', vstsRefspec, 'VSTS refspec')
+                stringParam('VSTSBranchOrCommit', vstsBranchOrCommit, 'VSTS commit hash')
```

Comment:
Does the casing of VSTSRefSpec and VSTSBranchOrCommit match the plugin's casing?